test/e2e/modal.test.js

```diff
@@ -3,6 +3,7 @@ const VlModalPage = require('./pages/vl-modal.page');
 const VlModalAutoOpenPage = require('./pages/vl-modal-auto-open.page');
 const { VlDatepicker } = require('vl-ui-datepicker').Test;
 const { VlInputField } = require('vl-ui-input-field').Test;
+const { Key } = require('selenium-webdriver');
```

Comment:
Ik zou ofwel die Key in de core steken (zoals de By), ofwel expliciet dependen op selenium-webdriver in de package.json dependencies (nu is het enkel transitief). Mijn voorkeur gaat uit naar het eerste.

Reply:
De dependency komt onrechtstreeks mee via de vl-ui-util, maar ik zal Key exposen via de core.

Reply:
@SanderKleykens omdat issue al resolveer was blijkbaar, ben ik het vergeten aan te passen. De vl-ui-core werd nog in deze PR toegevoegd, maar het vervangen niet. Ik heb dit rechtstreeks op de #master toegepast.